spec/acceptance/standard_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'spec_helper_acceptance'
+
+describe 'hashi_stack::repo class' do
+
+  context 'default parameters' do
+    # Using puppet_apply as a helper
+    it 'should work with no errors based on the example' do
+      pp = <<-EOS
+        class { 'hashi_stack::repo': }
+      EOS
+
+      # Run it twice and test for idempotency
+      expect(apply_manifest(pp).exit_code).to_not eq(1)
+      expect(apply_manifest(pp).exit_code).to eq(0)
```

Comment:
Typically we use `apply_manifest(pp, catch_failures: true)` the first time and `apply_manifest(pp, catch_changes: true)` the second time.

Reply:
i ll make those changes!